GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategoryByCospans.gi

```diff
@@ -376,6 +376,130 @@ InstallMethod( AsGeneralizedMorphismByCospan,
     
 end );
 
+####################################
+##
+## Constructors of lifts of exact functors
+##
+####################################
+
+##
+InstallMethod( AsGeneralizedMorphismByCospan,
+        [ IsCapFunctor, IsString, IsCapCategory, IsCapCategory ],
+
+  function( F, name, A, B )
```

Comment:
The arguments `A` and `B` seem to be superfluous since
they can be read off from the source and range of `F`
(`GeneralizedMorphismCategoryByCospans` is an attribute of a Cap category).